script/deploy/zksync/DeployLiFiDEXAggregator.zksync.s.sol

```diff
@@ -23,13 +23,15 @@ contract DeployScript is DeployScriptBase {
 
     function getConstructorArgs() internal override returns (bytes memory) {
         string memory path = string.concat(root, "/config/global.json");
-        string memory json = vm.readFile(path);
 
         address[] memory priviledgedUsers = new address[](1);
-        priviledgedUsers[0] = json.readAddress(".pauserWallet");
+        priviledgedUsers[0] = _getConfigContractAddress(path, ".pauserWallet");
```

Comment:
I recall we had the similar feedback for abstract PR. Shouldn't `_getConfigContractAddress` be only used for contracts not EOAs.

Has this been updated?

Reply:
these are comments on unrelated/untouched code by this PR. I suggest that we create a new PR for this. What do you think?

Reply:
good idea, let's do it